bgpd/bgp_rpki.c

```diff
@@ -273,6 +274,15 @@ static void print_record(const struct pfx_record *record, struct vty *vty)
 		record->max_len, record->asn);
 }
 
+static void print_record_by_asn(const struct pfx_record *record, void *data)
+{
+	struct rpki_for_each_record_arg *arg = data;
+	struct vty *vty = arg->vty;
+
+	if (record->asn == arg->as)
```

Comment:
is it possible to increment prefix_amount ?

Reply:
ah, sure, I'll update.